### DISCUSSION TITLE
Name on "View Assist Device Configuration" page

### DISCUSSION THREAD

**Discussion by mllockwood** [dinki/View-Assist#13] [General]
Does the name of the group entities on this page (https://github.com/dinki/View-Assist/wiki/View-Assist-device-configuration) may need to be modified/edited? 

In the template configuration section, you're using hypens between words, but in the group configuration you are using underscores.  It's right in the associated YouTube video though.

template:
  - sensor:
    - name: **ViewAssist-livingroom**

group:
  viewassist_satellites:
    name: View Assist Satellites
    entities:
      - sensor.**viewassist_livingroom**

**Comment by dinki** [comment]
Nice catch.  Thank you.  I've updated the wiki.
